build/source/driver/multi_driver.f90

```diff
@@ -211,23 +212,44 @@ program multi_driver
 integer(i4b)                     :: err=0                      ! error code
 character(len=1024)              :: message=''                 ! error message
 
+! subsetting the model run
+integer(i4b)                     :: strtHRU                    ! index of the starting hru for a subset run
+character(len=10)                :: strTMP                     ! local temp string
```

Comment:
Should we have a general local temp-string, or should we have a separate varaible for each temporary string that we might need? With a self-described name.